rust/crd/src/lib.rs

```diff
@@ -157,28 +173,27 @@ impl OpaCluster {
     }
 
     /// Retrieve and merge resource configs for role and role groups
-    pub fn resolve_resource_config_for_role_and_rolegroup(
+    pub fn merged_config(
         &self,
-        role: &OpaRole,
         rolegroup_ref: &RoleGroupRef<OpaCluster>,
-    ) -> Result<Resources<OpaStorageConfig, NoRuntimeLimits>, Error> {
+    ) -> Result<OpaConfig, Error> {
         // Initialize the result with all default values as baseline
-        let conf_defaults = OpaConfig::default_resources();
-
-        let role = match role {
-            OpaRole::Server => &self.spec.servers,
-        };
```

Comment:
**nit:** The whole code uses `OpaRole`. I would also keep it here.

Reply:
done https://github.com/stackabletech/opa-operator/pull/411/commits/bf1ed19e3a9bf785c8f8aafe949317cd71c83301